bigquery/docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
       - 127.0.0.1:9200:9200
   indexer:
     build:
-      # Context is project root, to pick up indexer_util
+      # Context is project root, to pick up indexer_util and dataset_config
       context: ../
       dockerfile: bigquery/Dockerfile
     environment:
```

Comment:
Default DATASET_CONFIG_DIR on line 15 should be ../dataset_config/platinum_genomes. Test it manually to make sure this works?